content/about/feature-stages.md

```diff
@@ -35,22 +35,26 @@ Below is our list of existing features and their current phases. This informatio
 | [Protocols: TCP](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/tcp_proxy)    | Alpha
 | [Protocols: WebSocket](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/websocket)      | Alpha
```

Comment:
This doesn't exist/work anymore in 0.8 alpha3, change it to Not Started maybe?

Reply:
Removed for now..will include after TCP and Websocket are fixed